src/kgcl_schema/grammar/parser.py

```diff
@@ -54,6 +58,15 @@ def parse_statement(input: str) -> Change:
 
     Return an instantiated dataclass object from model.kgcl_schema.
     """
+    regex = r'<http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+    uri = re.findall(regex, input)
+    if uri:
+        # curie = curie_from_iri(uri[0].replace("<", "").replace(">",""))
+        pref, i = parse_iri(uri[0].replace("<", "").replace(">",""))
+        pref = get_preferred_prefix(pref)
+        curie = curie_to_str(pref, i)
+        input = input.replace(uri[0], curie)
+
```

Comment:
It seems very hacky to replace URIs with CURIEs just so the KGCL parser works. Why is that necessary?

Reply:
When adding synonyms, apparently only CURIEs are expected and ehnce the conversion from URI => CURIE. Just a judgement call, there could be a better way of handling this. Ideas?

Reply:
I dont know how that parser works, but if CURIES are expected, wouldn't it be better to make sure the input is "valid kgcl" using a validator and not trying to hack anything? The user can be forced to provide CURIEs if that is what the spec wants..

In any case, I would for sure factor out the method `def replace_uris_with_curies_in_kgcl_command(input)` into its how, document it etc. Not convinced this is great but if its necessary to be parseable than so be it.